modules/virt-importing-vm-prerequisites.adoc

```diff
@@ -1,57 +1,29 @@
 // Module included in the following assemblies:
 // * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
-// * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
-[id='virt-importing-vm-prerequisites_{context}']
-= Prerequisites for importing a virtual machine
-
-ifdef::virt-importing-rhv-vm[]
-Importing a virtual machine from Red Hat Virtualization (RHV) into {VirtProductName} has the following prerequisites.
-
-== Red Hat Virtualization prerequisites
 
-The Red Hat Virtualization (RHV) environment has the following prerequisites for VM import:
-
-* Network:
+[id='virt-importing-vm-prerequisites_{context}']
```

Comment:
I know this was already here, but these should be double quotes if you don't mind updating.